Dockerfile

```
@@ -1,5 +1,5 @@
 # Building studio frontend
-FROM node:21.5.0-alpine AS generate-studio-frontend
+FROM node:20.11.0-alpine AS generate-studio-frontend
```

I wonder if we could just use the LTS tag of the alpine image:
```suggestion
FROM node:lts-alpine AS generate-studio-frontend
```

---

Nice suggestion! Yes, we can! 💯